tensorflow/lite/kernels/internal/BUILD

```diff
@@ -137,3 +137,42 @@ cc_library(
         ":compatibility",
     ],
 )
+
+cc_library(
+    name = "portable_tensor_utils",
+    srcs = [
+        "reference/portable_tensor_utils.cc",
+    ],
+    hdrs = [
+        "reference/portable_tensor_utils.h",
+        "reference/portable_tensor_utils_impl.h",
+    ],
+    copts = tflite_copts(),
+    deps = [
+        ":common",
+        ":compatibility",
+        ":cppmath",
+        "@gemmlowp",
+    ],
+)
+
+cc_library(
+    name = "tensor_utils",
+    srcs = [
+        "portable_tensor_utils.cc",
+        "tensor_utils.cc",
+    ],
+    hdrs = [
+        "portable_tensor_utils.h",
+        "tensor_utils.h",
+    ],
+    copts = tflite_copts(),
+    deps = [
+        ":common",
+        ":compatibility",
+        ":cppmath",
+        ":portable_tensor_utils",
+        "//tensorflow/lite/c:common",
+        "//tensorflow/lite/kernels:op_macros",
+    ],
+)
```

Comment:
let's collapse both the build targets into one. We will break parity with TfLite BUILD files in that process so we should document what we are doing in the comments.